modules/ROOT/pages/ai-azure.adoc

```diff
@@ -0,0 +1,215 @@
+
+A proxy is required if making `fetch` calls directly to the Azure OpenAI REST API as otherwise CORS errors arise. In general, using a proxy is highly recommended even if network errors did not occur. See the xref:ai-proxy.adoc[AI Proxy Server reference guide] for information on how to setup a proxy server for use with the {pluginname}.
+
+NOTE: The xref:ai-proxy.adoc[AI Proxy Server reference guide] is, as its name notes, a reference. There is no single proxy server setup that is right or correct for all circumstances and other setups may be better for your use-case.
+
+== String response
+
+This example demonstrates how to integrate the {pluginname} plugin with the https://learn.microsoft.com/en-us/azure/ai-services/openai/reference#chat-completions[Azure OpenAI Service Chat completions API] to generate a string response.
+
+[source,js]
+----
+// This example stores the API key in the client side integration. This is not recommended for any purpose.
+// Instead, an alternate method for retrieving the API key should be used.
+const AZURE_OPENAI_API_KEY = '<INSERT_API_KEY_HERE>';
+const AZURE_OPENAI_ENDPOINT = '<INSERT_ENDPOINT_URL_HERE>'; // e.g. https://{your-resource-name}.openai.azure.com/openai/deployments/{deployment-id}/chat/completions?api-version={api-version}
+
+const ai_request = (request, respondWith) => {
+  const azureOpenAiOptions = {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+      'Authorization': `Bearer ${AZURE_OPENAI_API_KEY}`
+    },
+    body: JSON.stringify({
+      temperature: 0.7,
+      max_tokens: 800,
+      messages: [{ role: 'user', content: request.prompt }],
+    })
+  };
+  respondWith.string((signal) => window.fetch(AZURE_OPENAI_ENDPOINT, { signal, ...azureOpenAiOptions })
+    .then(async (response) => {
+      if (response) {
+        const data = await response.json();
+        if (data.error) {
+          throw new Error(`${data.error.type}: ${data.error.message}`);
+        } else if (response.ok) {
+          // Extract the response content from the data returned by the API
+          return data?.choices[0]?.message?.content?.trim();
+        }
+      } else {
+        throw new Error('Failed to communicate with the Azure OpenAI API');
+      }
+    })
+  );
+};
+
+tinymce.init({
+  selector: 'textarea',  // Change this value according to your HTML
+  plugins: 'ai',
+  toolbar: 'aidialog aishortcuts',
+  ai_request
+});
+----
+
+== Streaming response
+
+This example demonstrates how to integrate the {pluginname} plugin with the https://learn.microsoft.com/en-us/azure/ai-services/openai/reference#chat-completions[Azure OpenAI Service Chat completions API] to generate streaming responses.
+
```

Comment:
```suggestion

include::partial$misc/admon-ai-proxy.adoc[]

```

Reply:
We only need to include this once per page.